src/olgm/herald/Feature.js

```diff
@@ -101,7 +101,7 @@ class FeatureHerald extends Herald {
 
     this.featureListener_ = new Listener(this.feature_.on('change', () => this.handleFeatureChange_()));
 
-    this.listener = new PropertyListener(this.feature_, null, 'geometry', (geometry, oldGeometry) => {
+    this.listener = new PropertyListener(this.feature_, null, this.feature_.geometryName_ ? this.feature_.geometryName_ : 'geometry', (geometry, oldGeometry) => {
```

Comment:
`geometryName_` is a private property.

You should use `this.feature_.getGeometryName()` instead, which also always returns the name (it would return 'geometry') so no need for an if.

Reply:
Ready